_format/1.1/index.md

```diff
@@ -2371,6 +2393,23 @@ An error object **MAY** have the following members:
     exists; if it doesn't, the client **SHOULD** simply ignore the pointer.
   * `parameter`: a string indicating which URI query parameter caused
     the error.
+  * <a id="error-objects-source-missing"></a>`missing`: an object indicating
+    that the error was caused by a missing profile, JSON object key (or array 
+    index), or URI query parameter. If present, this object **MUST** contain 
```

Comment:
I can't think of a use case for the missing array index. What am I missing?

Reply:
The coordinate example helps, thanks!

But I thought profiles only could add elements to the document structure (per 
https://jsonapi.org/format/1.1/#profile-keywords-and-aliases), which means they must have a name, and hence a JSON object key? Wouldn't the coordinate example be a *value* of an object key, and therefore `z` being absent from `[x,y,z]` would not result in a `missing` error since the key would be present, the value itself would just be invalid. In other words: `invalid`, not `missing`?

I'm probably still missing something, sorry.

Reply:
> Wouldn't the coordinate example be a _value_ of an object key, and therefore z being absent from [x,y,z] would not result in a missing error since the key would be present, the value itself would just be invalid.

It's both. I.e., the value is invalid because a key (index 2) is missing.

In other words, if a client client sends: 

```
{ "meta": { "homeBase": [x,y] }
```

then the server's error object would ideally contain `"pointer": "/meta/homeBase"`, whether it contains `"missing"` or not. But that just tells you that the value at `/meta/homeBase` had an error. (Optionally) supplementing the pointer with `"missing": { "key": "2" }` just makes it a bit more specific about what the error was.